tests/testthat/test-fbref.R

```diff
@@ -381,7 +398,9 @@ Sys.sleep(3)
 test_that("fb_league_stats() for players works", {
   testthat::skip_on_cran()
   testthat::skip_on_ci()
-  expected_player_shooting_cols <- c("Rk", "Player", "Nation", "Pos", "Squad", "Age", "Born", "Mins_Per_90", "Gls", "Sh_Standard", "SoT_Standard", "SoT_percent_Standard", "Sh_per_90_Standard", "SoT_per_90_Standard", "G_per_Sh_Standard", "G_per_SoT_Standard", "Dist_Standard", "FK_Standard", "PK", "PKatt", "xG_Expected", "npxG_Expected", "npxG_per_Sh_Expected", "G_minus_xG_Expected", "np:G_minus_xG_Expected", "Matches", "url")
+
+  expected_player_shooting_cols <- c("Rk", "Player", "Player_Href", "Nation", "Pos", "Squad", "Age", "Born", "MP_Playing Time", "Starts_Playing Time", "Min_Playing Time", "Mins_Per_90_Playing Time", "Gls", "Ast", "G+A", "G_minus_PK", "PK", "PKatt", "CrdY", "CrdR", "xG_Expected", "npxG_Expected", "xAG_Expected", "npxG+xAG_Expected", "PrgC_Progression", "PrgP_Progression", "PrgR_Progression", "Gls_Per 90 Minutes", "Ast_Per 90 Minutes", "G+A_Per 90 Minutes", "G_minus_PK_Per 90 Minutes", "G+A_minus_PK_Per 90 Minutes", "xG_Per 90 Minutes", "xAG_Per 90 Minutes", "xG+xAG_Per 90 Minutes", "npxG_Per 90 Minutes", "npxG+xAG_Per 90 Minutes", "Matches", "url")
```

Comment:
looks like FBref changed some of the columns here? (G minus xG is no longer on the current page, for example) we may consider changing `.rename_fb_cols()`